cmd/gin-repod/main.go

```diff
@@ -269,6 +270,12 @@ Options:
 	s.SetupServiceSecret()
 	s.SetupStores()
 
+	s.Handler = handlers.CORS(
+		handlers.AllowedHeaders([]string{"Accept", "Content-Type", "Authorization"}),
+		handlers.AllowedOrigins([]string{"*"}),
+		handlers.AllowedMethods([]string{"GET", "PUT", "POST", "DELETE"}),
+	)(s.Handler)
```

Comment:
Good idea to add CORS support, looks good, although I have to admit I am kinda tempted to just c&p the ~70 lines into our server code.